lib/cli/src/Cardano/CLI.hs

```diff
@@ -1388,6 +1398,15 @@ metadataOption = option txMetadataReader $ mempty
 txMetadataReader :: ReadM ApiTxMetadata
 txMetadataReader = eitherReader (Aeson.eitherDecode' . BL8.pack)
 
+-- | [--ttl=SECONDS]
+timeToLiveOption :: Parser (Maybe NominalDiffTime)
+timeToLiveOption = optional $ option diffTime $ mempty
+    <> long "ttl"
+    <> metavar "SECONDS"
+    <> help "Time-to-live value in seconds. Default is 2 hours."
+  where
+    diffTime = fromIntegral <$> (auto :: ReadM Int)
```

Comment:
This is slightly inconsistent with the `--sync-tolerance` option that is already parsing time. This option requires an explicit suffix to disambiguate the argument: `14s`. 

It relies on the `FromText` instance of `SyncTolerance` to do so, what about re-using that logic here too?

Reply:
OK I doubt that users would particularly like having to add `s` when no other units are supported, but I have changed it anyway.